src/js/pages/Teams.jsx

```diff
@@ -19,32 +19,31 @@ import { METHOD, useFetchData } from '../react-query/WeConnectQuery';
 // eslint-disable-next-line no-unused-vars
 const Teams = ({ classes, match }) => {
   renderLog('Teams');
-  const { apiDataCache, setAppContextValue, getAppContextValue } = useConnectAppContext();
-  const { allPeopleCache, allTeamsCache } = apiDataCache;
+  const { apiDataCache } = useConnectAppContext();
   const dispatch = useConnectDispatch();
 
-  const [showAllTeamMembers, setShowAllTeamMembers] = useState(true);
-  const [teamList, setTeamList] = useState([]);
+  const { setAppContextValue, getAppContextValue } = useConnectAppContext();
 
-  const teamListRetrieveResults = useFetchData(['team-list-retrieve'], {}, METHOD.GET);
-  useEffect(() => {
```

Comment:
While we are comparing approaches, please comment out instead of deleting the alternative so it is easier to discuss.

Reply:
Yes, I tired to do that, again a reversion and cherry pick mistake.